pyproject.toml

```diff
@@ -119,8 +119,23 @@ fix = true
 include = ["*.py"]
 
 [tool.ruff.lint]
-select = ["W", "E", "F"]
+select = ["D", "W", "E", "F"]
 ignore = [
+    "D100", # Missing docstring in public module"
+    "D101", # Missing docstring in public class"
+    "D102", # Missing docstring in public method
+    "D103", # Missing docstring in public function
+    "D104", # Missing docstring in public package"
+    "D105", # Missing docstring in magic method"
+    "D107", # Missing docstring in `__init__`"
+    "D200", # One-line docstring should fit on one line"
+    "D203", # one-blank-line-before-class and `no-blank-line-before-class` (D211) are incompatible.
```

Comment:
I'm proposing D211 over D203. What I care most about is that we have a rule enforcing consistency.

Reply:
Removed the line